src/webgpu/api/operation/rendering/depth_clip_clamp.spec.ts

```diff
@@ -176,7 +172,7 @@ have unexpected values then get drawn to the color buffer, which is later checke
         topology: 'point-list',
         // `|| undefined` is a workaround for Chromium not allowing `false` here
         // when the feature is unavailable.
-        clampDepth: clampDepth || undefined,
+        unclippedDepth: unclippedDepth || undefined,
```

Comment:
```suggestion
        unclippedDepth,
```

Reply:
This looks wrong to me.  Shouldn't it be  `unclippedDepth: unclippedDepth` ?

Reply:
Filed https://github.com/gpuweb/cts/issues/2460

Reply:
Nevermind.  I need to learn JS better.